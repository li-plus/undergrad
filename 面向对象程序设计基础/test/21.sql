create database oop;
use oop;
create table oop_info(stu_id int not null, stu_name char, primary key(stu_id)); 
load data infile '1_in_file' into table oop_info(stu_id, stu_name);
select * from oop_info; 
/********************** Sample Output *************************
stu_id  stu_name        
2018011343      a       
2018011344      b       
2018011345      c    
**************************************************************/
drop database oop;