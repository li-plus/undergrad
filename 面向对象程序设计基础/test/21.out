stu_id  stu_name        
2018011343      a       
2018011344      b       
2018011345      c    
